python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -859,7 +937,109 @@ def massage(bb, tensor):
         return relax.Tuple([output, present])
 
 
-# pylint: enable=invalid-name, import-self, len-as-condition, unused-argument, too-many-lines
+class Identity(OnnxOpConverter):
+    """Converts an onnx Identity node into an equivalent Relax expression."""
+
+    @classmethod
+    def _impl_v1(cls, bb, inputs, attr):
+        return inputs[0]
+
+
+class Resize(OnnxOpConverter):
+    """Converts an onnx Resize node into an equivalent Relax expression."""
+
+    @classmethod
+    def _impl_v18(cls, bb, inputs, attr):
+        # Extract the many attributes of resize.
+        coord_mode = attr.get("coordinate_transformation_mode", b"half_pixel").decode("ascii")
+        cubic_coeff_a = attr.get("cubic_coeff_a", -0.75)
+        exclude_outside = attr.get("exclude_outside", 0)
+        extrapolation_value = attr.get("extrapolation_value", 0.0)
+        mode = attr.get("mode", b"nearest").decode("ascii")
+        rounding_method = attr.get("nearest_mode", b"round_prefer_floor").decode("ascii")
+
+        # Adapt attributes to fit TVM definition.
+        if mode == "nearest":
+            mode = "nearest_neighbor"
+
+        # Unpack inputs.
+        x = inputs[0]
+        roi = inputs[1]
+        scales = inputs[2]
+        sizes = inputs[3]
+        ndims = len(x.struct_info.shape)
+        assert ndims == 4, "Only resize2d is currently supported."
+
+        assert (
+            scales is None or sizes is None
+        ), "Only one of scales and sizes can be provided in Resize."
+
+        # Define relax implementation.
+        if roi is not None:
+            roi = relax.op.concat(
+                [
+                    relax.op.strided_slice(roi, axes=[0], begin=[2], end=[ndims]),
+                    relax.op.strided_slice(roi, axes=[0], begin=[ndims + 2], end=[2 * ndims]),
+                ],
+                axis=0,
+            )
```

Comment:
Would this require legalization? Not sure if this can be directly passed to emit_te.